agent/metadata_service.go

```diff
@@ -103,6 +104,10 @@ Return fake data for programs that depend on data from the metadata service.
 These fields are constructed to be obviously wrong and would never be found in the
 production environment.
 */
+func (mds *metadataService) getServices(w http.ResponseWriter, r *http.Request) {
+	fmt.Fprint(w, "fake-meta-data")
```

Comment:
technically, /latest redirects to /latest/ and then returns "meta-data" because its listing the available paths, so the fake- here isn't needed